src/components/Menu/MenuTitle/MenuTitleConstants.ts

```diff
@@ -0,0 +1,5 @@
+export enum MenuTitleCaptionPositions {
```

Comment:
It's convention to name enums as sungular:

```suggestion
export enum MenuTitleCaptionPosition {
```